config.json

```diff
@@ -90,14 +90,14 @@
         "name": "Calculator Conundrum",
         "uuid": "9e62feec-c2c3-4fd6-94f5-0574cc65447d",
         "concepts": [
-          "exception-handling"
+          "exceptions"
         ],
         "prerequisites": [
-          "conditionals",
-          "switch-case",
-          "basics"
+          "basics",
```

Comment:
You can omit `basics` as this is indirectly references via the other two prerequisites (it's fine to leave it in if you want)

```suggestion
```